bin/assembleCoadd.py

```diff
@@ -20,7 +20,34 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
+from __future__ import print_function
 import lsst.pex.logging as pexLog
-from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask
+from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask, SafeClipAssembleCoaddTask
+import sys
+import argparse
 
-AssembleCoaddTask.parseAndRun()
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--legacyCoadd', action='store_true', default=False)
+parser.add_argument('-h','--help', action='store_true',default=False)
+result, extra = parser.parse_known_args(sys.argv)
+
+if result.help:
+    legacy_message = '''
+Meta Arguments:
+  --legacyCoadd         An option to run with the original assembleCoadd task which will
+                        not attempt to do any safe clipping.  If run, the config option
+                        'config.measurement.plugins["base_PixelFlags"].any = [] must be set
+'''
+    print(legacy_message)
+    SafeClipAssembleCoaddTask.parseAndRun(args=['--help']+extra[1:])
+
+if result.legacyCoadd:
+    legacy_message = '''
+    Remember, If you use the legacy adder you must set
+    config.measurement.plugins['base_PixelFlags'].any = []
+    so that pixel flags will not look for a non existent mask plane
+    '''
+    print(legacy_message)
+    AssembleCoaddTask.parseAndRun(args=extra[1:])
+else:
+    SafeClipAssembleCoaddTask.parseAndRun(args=extra[1:])
```

Comment:
It's unfortunate that both `assembleCoadd.py --help` and `assembleCoadd.py` produce help messages, but they aren't the same. Is it possible to catch usage errors from these tasks and add your "meta arguments" help block to the output?